crates/script/src/execute.rs

```diff
@@ -420,15 +421,16 @@ impl PreSimulationState {
             }
 
             shell::println("Traces:")?;
-            for (kind, trace) in &result.traces {
+            for (kind, mut trace) in result.traces.clone() {
                 let should_include = match kind {
```

Comment:
(follow-up) we need to unify this logic of verbosity -> should_include between cast/test/coverage/script...

Reply:
Opened follow-up ticket here: https://github.com/foundry-rs/foundry/issues/8339